### DISCUSSION TITLE
How to use reset through neogit?

### DISCUSSION THREAD

**Discussion by emersonfbarros** [NeogitOrg/neogit#1207] [Q&A] [answered]
I open the reset popup by typing X (capital x) while the cursor is on the printscreen commit (test), I select the s (soft) option, but nothing happens. Nothing happens when typing the other options either, like m, h, etc. Am I doing something wrong or is this a bug?

![image](https://github.com/NeogitOrg/neogit/assets/72613326/fdc9ea8e-70f3-4f01-b90d-6be379e2131c)

**Comment by CKolkey** [comment, marked as answer]
This is slightly confusing - it resets _to_ the selected commit. So if you want to do a soft reset _of_ that commit, you will need to reset to it's parent. I think that's what you mean, yea?

**Reply by emersonfbarros** [reply]
exactly. I got confused about which commit to apply the reset to. thanks for clarifying!

**Reply by CKolkey** [reply]
Sure thing. Best way to remember it is that it's the same as with the CLI :)
